src/Servers/Kestrel/test/InMemory.FunctionalTests/ChunkedResponseTests.cs

```diff
@@ -421,15 +421,21 @@ public async Task ChunksCanBeWrittenManually()
             }
         }
 
-        [Fact]
-        public async Task ChunksWithGetMemoryBeforeFirstFlushStillFlushes()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
```

Comment:
I'd rather have a few tests explicitly for calling/not calling StartAsync rather than adding variations to existing tests.

Reply:
Are you asking for me to write new tests that try different StartAsync/NonStartAsync scenarios, or are you asking that I split the Theory into two different tests?

I'm assuming you are asking for both when appropriate.